### PR TITLE
stoa0040_more files

### PR DESCRIPTION
**Pull request by AlisonBabeu**
Updated cts and bibliographic description of more Augustine files

### CODE REVIEW

File: data/stoa0040/stoa046/stoa0040.stoa046.opp-lat1.xml [hunk 1]
```diff
@@ -40,36 +43,36 @@
...
-                <date>1902</date>
+                <publisher>F. Tempsky</publisher>
+                <publisher>G. Freytag</publisher>
+                <pubPlace>Prague</pubPlace>
+                <pubPlace>Vienna</pubPlace>
+                <pubPlace>Leipzig</pubPlace>
+                <date>1904</date>
               </imprint>
-              <biblScope unit="volume">42</biblScope>
+              <biblScope unit="volume">Sectio 8, Pars 2</biblScope>
```
Comment (lcerrato) [inline data/stoa0040/stoa046/stoa0040.stoa046.opp-lat1.xml:69]: Probably should be tagged as Latin?

File: data/stoa0040/stoa042/stoa0040.stoa042.opp-lat1.xml [hunk 2]
```diff
@@ -40,28 +42,31 @@
...
-                <pubPlace>Wien, Österreich</pubPlace>
+                <publisher>F. Tempsky</publisher>
+                <publisher>G. Freytag</publisher>
+                <pubPlace>Prague</pubPlace>
+                <pubPlace>Vienna</pubPlace>
+                <pubPlace>Leipzig</pubPlace>
                 <date>1894</date>
               </imprint>
-              <biblScope unit="volume">28.1</biblScope>
+              <biblScope unit="volume" xml:lang="lat">Sectio III, Pars 1-2</biblScope>
```
Comment (lcerrato) [inline data/stoa0040/stoa042/stoa0040.stoa042.opp-lat1.xml:63]: Latin? Also, I don't think that the Roman numerals are used elsewhere in this set. (I think some have Arabic numbers and some have Roman numerals.)

File: data/stoa0040/stoa042/stoa0040.stoa042.opp-lat1.xml [hunk 3]
```diff
@@ -1,8 +1,10 @@
 <?xml version="1.0" encoding="UTF-8"?><?xml-model href="http://www.stoa.org/epidoc/schema/latest/tei-epidoc.rng" schematypens="http://relaxng.org/ns/structure/1.0"?><TEI xmlns="http://www.tei-c.org/ns/1.0">
-<teiHeader>
+<teiHeader xml:lang="ENG">
```
Comment (lcerrato) [inline data/stoa0040/stoa042/stoa0040.stoa042.opp-lat1.xml:2]: lowercase eng

File: data/stoa0040/stoa070/stoa0040.stoa070.opp-lat1.xml [hunk 4]
```diff
@@ -40,27 +42,31 @@
...
-                <pubPlace>Wien, Österreich</pubPlace>
+                <publisher>F. Tempsky</publisher>
+                <publisher>G. Freytag</publisher>
+                <pubPlace>Prague</pubPlace>
+                <pubPlace>Vienna</pubPlace>
+                <pubPlace>Leipzig</pubPlace>
                 <date>1894</date>
               </imprint>
-              <biblScope unit="volume">28.1</biblScope>
+              <biblScope unit="volume" xml:lang="lat">Sectio III, Pars 1-2</biblScope>
```
Comment (lcerrato) [inline data/stoa0040/stoa070/stoa0040.stoa070.opp-lat1.xml:63]: same as above: language and Roman numerals

File: data/stoa0040/stoa057a/stoa0040.stoa057a.opp-lat1.xml [hunk 5]
```diff
@@ -40,36 +43,36 @@
...
-                <date>1902</date>
+                <publisher>F. Tempsky</publisher>
+                <publisher>G. Freytag</publisher>
+                <pubPlace>Prague</pubPlace>
+                <pubPlace>Vienna</pubPlace>
+                <pubPlace>Leipzig</pubPlace>
+                <date>1904</date>
               </imprint>
-              <biblScope unit="volume">42</biblScope>
+              <biblScope unit="volume">Sectio 8, Pars 2</biblScope>
```
Comment (lcerrato) [inline data/stoa0040/stoa057a/stoa0040.stoa057a.opp-lat1.xml:69]: same as others

File: data/stoa0040/stoa056c/stoa0040.stoa056c.opp-lat1.xml [hunk 6]
```diff
@@ -40,35 +43,36 @@
...
-                <date>1902</date>
+                <publisher>F. Tempsky</publisher>
+                <publisher>G. Freytag</publisher>
+                <pubPlace>Prague</pubPlace>
+                <pubPlace>Vienna</pubPlace>
+                <pubPlace>Leipzig</pubPlace>
+                <date>1904</date>
               </imprint>
-              <biblScope unit="volume">42</biblScope>
+              <biblScope unit="volume">Sectio 8, Pars 2</biblScope>
```
Comment (lcerrato) [inline data/stoa0040/stoa056c/stoa0040.stoa056c.opp-lat1.xml:69]: same as others

File: data/stoa0040/stoa056a/stoa0040.stoa056a.opp-lat1.xml [hunk 7]
```diff
@@ -40,35 +43,36 @@
...
-                <date>1902</date>
+                <publisher>F. Tempsky</publisher>
+                <publisher>G. Freytag</publisher>
+                <pubPlace>Prague</pubPlace>
+                <pubPlace>Vienna</pubPlace>
+                <pubPlace>Leipzig</pubPlace>
+                <date>1904</date>
               </imprint>
-              <biblScope unit="volume">42</biblScope>
+              <biblScope unit="volume">Sectio 8, Pars 2</biblScope>
```
Comment (lcerrato) [inline data/stoa0040/stoa056a/stoa0040.stoa056a.opp-lat1.xml:69]: same as others

**Review by lcerrato — CHANGES_REQUESTED** [review]

File: data/stoa0040/stoa042/stoa0040.stoa042.opp-lat1.xml (hunk 2 above)
Comment (AlisonBabeu) [reply data/stoa0040/stoa042/stoa0040.stoa042.opp-lat1.xml:63]: I"m pretty sure I have been totally inconsistent across these files. What do you think is best @lcerrato that we have Section 3, Part 1-2,  or Sectio 3, Pars 1-2, and then just add the language tag? I do think you are right that I haven't included Roman numerals before. I like to be consistently inconsistent!

File: data/stoa0040/stoa042/stoa0040.stoa042.opp-lat1.xml (hunk 3 above)
Comment (AlisonBabeu) [reply data/stoa0040/stoa042/stoa0040.stoa042.opp-lat1.xml:2]: oh my, how did I do that?

File: data/stoa0040/stoa042/stoa0040.stoa042.opp-lat1.xml (hunk 2 above)
Comment (lcerrato) [reply data/stoa0040/stoa042/stoa0040.stoa042.opp-lat1.xml:63]: @AlisonBabeu 
I think you just look at the rest of this collection and decide the best approach. Where it's Latin, I'm sure there are other Latin designations here. I'm sure that we use different designations throughout. 
I would lean towards keeping it as the book reads Latin + Roman numerals for ease of presentation. Just be sure the set is the same throughout.
(Note that the CTS files read **Sectio 3, Pars I-II** and this is the reverse **Sectio III, Pars 1-2**

Comment (AlisonBabeu) [reply data/stoa0040/stoa042/stoa0040.stoa042.opp-lat1.xml:63]: hi @lcerrato . Good plan. I will try to standardize these files and do a final pass through Augustine after I finish the last files.  I  think you are right and we should go with Latin and Roman numerals. What appears on the books seems to typically be Sect. VII Pars II, so they don't even spell out section. I do think I will stick with Arabic numbers for the volume numbers though, even though they also list those using Roman numerals.

**Review by lcerrato — APPROVED** [review]

**Pull request merged**